pkgs/build-support/go/module.nix

```diff
@@ -276,7 +276,7 @@ let
       export GOFLAGS=''${GOFLAGS//-trimpath/}
 
       for pkg in $(getGoDirs test); do
-        buildGoDir test "$pkg"
+        buildGoDir test -vet=off "$pkg"
```

Comment:
Not the place to pass flags. `buildGoDir` takes only 2 params, `cmd` and `dir`.

This looks like the place where test flags should be added:

https://github.com/NixOS/nixpkgs/blob/3346909badca499ac5066cef16e5a620101bc339/pkgs/build-support/go/module.nix#L214-L216